src/joystick/windows/SDL_rawinputjoystick.c

```diff
@@ -47,7 +47,7 @@
 #ifdef HAVE_XINPUT_H
 #define SDL_JOYSTICK_RAWINPUT_XINPUT
 #endif
-#ifdef SDL_WINDOWS10_SDK
+#ifdef HAVE_WINDOWS_GAMING_INPUT_H
```

Comment:
Just to say that I really like having this macro instead. Makes more sense semantically as this is really the one feature it needs here